keyboards/ai03/lunar/keymaps/muzfuz/readme.md

```diff
@@ -0,0 +1,8 @@
+# muzfuz's keymap for Lunar
+
+```shell
+make ai03/lunar:muzfuz:dfu
+dfu-programmer atmega32u4 erase --force && \
```

Comment:
If you have the above, you shouldn't need the rest.  

If you want to leave it in here, that's fine,  though I'd say use a different section/block for that, then.

Reply:
Cheers, that makes sense :) I've pushed that change.